collective/trustedimports/tests.py

```diff
@@ -34,7 +35,7 @@ def setUp(test=None):
     MSI('sets').declarePublic('Set')
     newSecurityManager(None, None)
     xmlconfig.file('configure.zcml', collective.trustedimports, ) #context=configurationContext)
-
+    os.environ["SAFEIMPORTS_URL_BLACKLIST"] = "https://www.w3schools.com/Xml/tempconvert.asmx?WSDL;http://www.dneonline.com/*.asmx?WSDL"
```

Comment:
this is for a specific test so it should go there. it also needs to be removed afterwards